system/Validation/Validation.php

```diff
@@ -295,7 +307,14 @@ protected function processRules(string $field, ?string $label, $value, $rules =
                     $value = json_encode($value);
                 }
 
-                $this->errors[$field] = $error ?? $this->getErrorMessage($rule, $field, $label, $param, $value);
+                $param                = ($param === false) ? '' : $param;
+                $this->errors[$field] = $error ?? $this->getErrorMessage(
```

Comment:
Can we add a line in between? The empty space between `$param` and `=` is quite much.

Reply:
Done.